### PR TITLE
Create For awardees.md

### PR DESCRIPTION
**Pull request by kategarklavs**
- Common forms and checklists are featured on the **Events/resources** page — OK to link to that instead of https://www.nsf.gov/eng/iip/sbir/Forms/index.jsp?
- For the Phase I and II general grant conditions, I didn't see md files so I linked to the existing pages — is that OK by you?

### CODE REVIEW

**Comment by OpenGlobe** [comment]
@kategarklavs I think I caught anything that was under that SBIR directory, and used the relative path structure for the md files.

@line47 can you take a look and see if this looks correct?

File: pages/grantees/For awardees.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Phase I
+
+Information for current Phase I awardees
+
+- [Phase I general grant conditions](https://www.nsf.gov/awards/managing/special_conditions.jsp)
+- [New awardee kit]({{ site.baseurl }}/assets/files/grantee-files/NSF-SBIR-STTR-PhaseIRespOpp.pdf) 
+- [CAAR Videos for Phase I Awardees: Financial Capability, Accounting, and Phase II Budgeting](https://www.youtube.com/playlist?list=PLGhBP1C7iCOmI1p5UtqYCXzmUL9SzSApv)
+- [Phase I reporting requirements]({{ site.baseurl }}/grantees/reporting/)
+- [How to apply for a Phase II award]({{ site.baseurl }}/grantees/how-to-apply-phaseii/)
```
Comment (line47) [inline pages/grantees/For awardees.md]: This link doesn't work

File: pages/grantees/For awardees.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Phase II 
+
+Information for current Phase II awardees
+
+- [Phase II general grant conditions](https://www.nsf.gov/awards/managing/special_conditions.jsp)
+- [Supplemental funding opportunies]({{ site.baseurl }}/grantees/supplement/)
```
Comment (line47) [inline pages/grantees/For awardees.md]: This link doesn't work

File: pages/grantees/For awardees.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- [Phase I reporting requirements]({{ site.baseurl }}/grantees/reporting/)
+- [How to apply for a Phase II award]({{ site.baseurl }}/grantees/how-to-apply-phaseii/)
+
+## Phase II 
+
+Information for current Phase II awardees
+
+- [Phase II general grant conditions](https://www.nsf.gov/awards/managing/special_conditions.jsp)
+- [Supplemental funding opportunies]({{ site.baseurl }}/grantees/supplement/) 
+- [Phase II reporting requirements]({{ site.baseurl }}/grantees/reporting2/)
```
Comment (line47) [inline pages/grantees/For awardees.md]: This link doesn't work

**Comment by gemfarmer** [comment]
This PR is waiting on some url clean up.

**Comment by gemfarmer** [comment]
@kategarklavs 
Things that should probably change before this issue is merged:
- [ ] updated all links that have a base url path of `/grantees/` to `/awardees/`. After doing links might not work yet because some of those pages don't exist yet on `dev`, but they will.
- [ ] Change `For awardees.md` to `for-awardees.md`

**Pull request closed without merging**